packages/cfd/src/Containers/jurisdiction-modal.tsx

```diff
@@ -0,0 +1,382 @@
+import React from 'react';
```

Comment:
Why are you moving around Jurisdiction components?

Reply:
@hirad-rewok please clarify 🙏

Reply:
The jurisdiction modal in the main folder of containers is the outdated one. Amina's refactored jurisdiction is in the jurisdiction folder in the containers folder and that's why I have changed the path of jurisdictions component.